caravel/assets/javascripts/SqlLab/components/DatabaseSelect.jsx

```diff
@@ -0,0 +1,51 @@
+const $ = window.$ = require('jquery');
+import React from 'react';
+import Select from 'react-select';
+
+class DatabaseSelect extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      databaseLoading: false,
+      databaseOptions: [],
+      databaseId: null,
+    };
+  }
+  componentWillMount() {
+    this.fetchDatabaseOptions();
+  }
+  changeDb(db) {
+    const val = (db) ? db.value : null;
+    this.setState({ databaseId: val });
+    this.props.onDatabaseSelected(db);
+  }
+  fetchDatabaseOptions() {
+    this.setState({ databaseLoading: true });
+    const url = '/databaseasync/api/read?_flt_0_expose_in_sqllab=1';
+    $.get(url, (data) => {
```

Comment:
i think we should use a smaller lib for ajax/get/post requests. there is a polyfill for the window.fetch standard. 
https://www.npmjs.com/package/whatwg-fetch

it's only ~3kb gzipped compared to ~70kb to include jquery on the page.